tox.ini

```diff
@@ -10,11 +10,14 @@ setenv =
 usedevelop = False
 install_command =
   pip install {opts} {packages}
-deps = pytest
+deps =
+  pytest
+  pytest-xdist
+  pytest-sugar
 allowlist_externals = mkdir
 commands =
   mkdir -p test-results/
-  pytest --junitxml=test-results/testenv-{envname}.xml {posargs}
+  pytest -n auto --junitxml=test-results/testenv-{envname}.xml {posargs}
```

Comment:
Added [pytest-xdist](https://pytest-xdist.readthedocs.io/en/latest/#pytest-xdist) plugin and the `-n auto` key to the pytest command to speed up test execution. _(Test execution time has decreased visibly)_

Unfortunately, **coverage.py** is not stable with `pytest-xdist` enabled. So for **coverage** `xdist` is not used.

Reply:
Actually... one nitpick, pls use full name of the flag: `--numprocesses`. Actually... isn't `auto` the default? 🤔

Reply:
> So for coverage xdist is not used.

Hmm actually... probably you can then replace `{[testenv]deps}` with just `pytest` + `pytest-sugar` in `[testenv:coverage]`... but that's a very minor thing. But indeed AFAICS `xdist` isn't used in `tox -e coverage`

Reply:
I suggest that "improving" `tox.ini` should be a separate task.

I just added `pytest-xdist` because the tests take time. You can take that out, too, if you are confused about it.

Reply:
No problem, let's leave as is for the moment